docs/src/main/asciidoc/kafka/kafka_overview.adoc

```diff
@@ -1016,3 +1016,58 @@ public KafkaBinderHealth kafkaBinderHealthIndicator() {
     };
 }
 ```
+
+[[custom-kafka-binder-health-indicator-example]]
+=== Custom kafka Binder Health Indicator Example
+
+Here is the pseudo-code for writing a custom Kafka binder HealthIndicator. In this example, we try to override the binder provided Kafka HealthIndicator by specifically checking first for cluster connectivity and then followed by topic-related issues.
+
+
+1. First we need create a custom implementation of the `KafkaBinderHealth` interface.
+
+```
+public class KafkaBinderHealthImplementation implements KafkaBinderHealth {
+    @Value("${spring.cloud.bus.destination}")
+    private String topic;
+    private final AdminClient client;
+
+    public KafkaBinderHealthImplementation(final KafkaAdmin admin) {
+		// More about configuring Kafka
+		// https://docs.spring.io/spring-kafka/reference/html/#configuring-topics
+        this.client = AdminClient.create(admin.getConfigurationProperties());
+    }
+
+    @Override
+    public Health health() {
+        if (!checkBrokersConnection()) {
+            logger.error("Error when connect brokers");
+        }
+		if (!checkTopicConnection()) {
+			logger.error("Error when trying to connect with specific topic");
+		}
+        return Health.up().build();
```

Comment:
The code does not return a down status if the above calls return `false.` Is that intended? Instead of logging the error, why don't you return `Health.down()..`?